### ISSUE TITLE
Handling duplicates 

### ISSUE THREAD

**Issue by Sfomalley** [PRiSMA-Study/PRISMA-Public#44]
https://github.com/PRiSMA-Study/PRISMA-Public/blob/e50fb738efbe51ea633bc854902cec39bedfdf8a/PRISMA-Maternal-Constructed-Outcomes/Infection/Maternal-Infection-Variables.R#L108

Checking I understand correctly: 
If there's a duplicate, you will keep the first form according to the date, and drop the subsequent forms?

If so:
In the case where there are two entries for the same form:
    #1 is a default date value because the form wasn't really completed with real data
    #2 the form was filled for real with a real date during PRISMA

would the current handling of duplicates preferentially choose the form #1, since default values will always be before PRISMA dates?
Is there a way to choose the first non-default date?

I appreciate your thoughts as I learn how to deal with duplicates myself!!

**Comment by sloisate2** [comment]
Great question!! I made this mistake one too many times before I finally figured it out haha 

In a previous processing script (not in this code) I replaced all default value dates with NAs for this purpose. This way, the NAs won't get in the way of duplicates. In the event, however I would want to use the default date as an indicator, I wouldn't want to have it completely deleted from the dataset. To get around this, I usually make a copy of the date variable so I end up with the following: 

visit_date_var1 = reported date (including default values dates) 
visit_date_var2 (newly generated) = reported date (with default value dates replaced with NA) 

if you use the visit_date_var2 variable in the scenario you outlined, option #2 would always be selected
